src/jobs/emailSequence.ts

```diff
@@ -3,12 +3,12 @@ import MailerService from '../services/mailer';
 
 export default class EmailSequenceJob {
 
-  public async handler(job, done): Promise<any> {
+  public async handler(job, done): Promise<void> {
     console.log('✌️ Email Sequence Job triggered!');
     try {
-      const { email, name } = job.data;
+      const { email, name }: { [key: string]: string } = job.data;
       const mailerServiceInstance = Container.get(MailerService);
-      await mailerServiceInstance.StartEmailSequence(email, name);
+      await mailerServiceInstance.StartEmailSequence(email, name as any); // Here you are passing name but in the function you spect user
```

Comment:
What do you think of this?

Reply:
That is a bug, it should be: 
`mailerServiceInstance.StartEmailSequence('WelcomeSequence', { email, name})`

Reply:
Would you mind in making the fix?

Reply:
Sure